### PR TITLE
add content to intro.md 'what is js?'

### PR DESCRIPTION
**Pull request by s-kngstn**
Hi Anthony, I saw your YouTube video on Open Source contributions and I felt compelled to help with this project. I added some content for the JavaScript intro page. 

I hope to contribute more to this project when I am free. Thanks for getting this project up and running.

All the best. Cheers!

-Sam

### CODE REVIEW

**Review by Harry-Hopkinson — APPROVED** [review]

**Comment by Harry-Hopkinson** [comment]
I did the same sort of thing in this pull request - https://github.com/anthonyshew/devtionary/pull/46. I will delete my What is Javascript section so your changes gets implemented.

**Comment by anthonyshew** [comment]
Hi, @s-kngstn! Thanks for the kind words and wanting to help out!

The goal that we have is to make things palatable for beginners and experts alike.

With that said, some of what you have written as an introductory level explanation to JavaScript is more geared towards intermediate or even expert level knowledge as far as programming goes.

Do you think you could edit your explanations some to try to make it more simple for a newcomer to programming to understand in simpler terms what JavaScript is?

Thanks! 😁 And thank you again for wanting to contribute!

**Comment by s-kngstn** [comment]
I can certainly refactor this section to make it more palatable for new comers and regular users alike. I will work on this some more within the week to cater to everyone @anthonyshew

**Comment by s-kngstn** [comment]
I have updated the intro section to cater to a wider audience and abstracted away some of the technical jargon from my original commit. Hopefully this is better suited to the section than before. I am looking forward to contributing more in the future and hope this first commit has helped provide a bit more content for the web site in general.

Hope you're all having a good weekend.

Cheers,

Sam
